DNN Platform/Website/Providers/DataProviders/SqlDataProvider/09.11.00.SqlDataProvider

```diff
@@ -51,6 +51,17 @@ BEGIN
 END
 GO
 
+/*    Install the Telerik Removal extension                 */
+/************************************************************/
+
+IF NOT EXISTS (SELECT 1 FROM {databaseOwner}[{objectQualifier}Packages] WHERE [Name] = 'TelerikRemoval')
+	INSERT INTO {databaseOwner}[{objectQualifier}Packages] (
+		[PortalID], [Name], [FriendlyName], [PackageType], [Version], [CreatedByUserID], [CreatedOnDate]
+	)
+	VALUES (
+		NULL, 'TelerikRemoval', 'Telerik Removal', 'Module', '9.11.0', -1, GETDATE()
+	)
+GO
```

Comment:
This seems like it's going to cause problems. We're assuming this in place to trick DNN into running the "upgrade" logic, but the other change I suggested should let us avoid needing to trick DNN at all.

```suggestion
```